src/libs/colors.js

```diff
@@ -17,7 +17,8 @@ const baseColors = {
   danger: '#db3214',
   light: '#e9ecef', // Used as header background color, lightened for background of cells, panels, etc.
   dark: '#333f52', // Used as text color, menu background (lightened), selected background (lightened)
-  grey: '#808080'
+  grey: '#808080',
+  disabled: '#6d6e70' //Used for disabled components at a 0.5 opacity.
```

Comment:
This will require testing with all the white-label color schemes (and possibly defining custom values for them). If there is any way to solve this problem without adding to ALL_COLORS, I highly suggest that you do that.

FYI @kyuksel

Reply:
I am OK with a disabled color being introduced as long as all the white-label sights are examined to see if they need to override the value for their theme.

Reply:
I checked the affected links on `Cloud Environments` page across [all brands](https://github.com/DataBiosphere/terra-ui/blob/dev/src/libs/brand-utils.js#L6) and they wouldn't be adversely impacted by this change but `Link` component is used in so many places that this would be a high-risk change (unhappy brand customers) and may require some sort of bug bashing with several folks knowledgeable about various areas of the UI since we don't currently have a way to automate these checks.

Speaking of automation, @cahrens would [your WIP](https://github.com/DataBiosphere/terra-ui/pull/3432) take care of the testing for this change? If yes, how close do you think it is?

Reply:
@kyuksel no it would not. It only checks accessibility on 2 pages-- billing profiles and workspace dashboard. It doesn't do anything like troll through the website… it just checks the currently visible HTML when you call the verify method.